pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -563,6 +563,10 @@ func (s *Installer) createValuesOverride(bundle releasev1alpha1.TinkerbellBundle
 						"name":  "prometheus_server",
 						"value": ":2213",
 					},
+					{
+						"name":  "lb_class_only",
```

Comment:
It would be useful and helpful for future us (and other EKSA contributors) to have a code comment here that explains the business logic for this. The why this is being set. You PR description alludes to what i think is the why but it doesn't quite hit it for me. This is the business logic from my perspective, feel free to disagree or add or subtract from it. 

```Go
// The Tinkerbell stack need a load balancer to work properly.
// We bundle Kube-vip in, as the load balancer, when we deploy the stack.
// We don't want this load balancer to be used by any other workloads.
// It allows us greater confidence in successful lifecycle events for the Tinkerbell stack, amongst other things.
// Also, the user should be free from Tinkerbell stack constraints
// and free to deploy a load balancer of their choosing and not be coupled to ours. 
```